src/winhello.c

```diff
@@ -285,29 +284,56 @@ pack_cose(WEBAUTHN_COSE_CREDENTIAL_PARAMETER *alg,
 }
 
 static int
-pack_cred_ext(WEBAUTHN_EXTENSIONS *out, WEBAUTHN_CRED_PROTECT_EXTENSION_IN *prot,
-    fido_cred_ext_t *in)
+pack_cred_ext(WEBAUTHN_EXTENSIONS *out, fido_cred_ext_t *in)
 {
 	WEBAUTHN_EXTENSION *e;
+	WEBAUTHN_CRED_PROTECT_EXTENSION_IN *p;
+	BOOL *b;
+	size_t n = 0, i = 0;
 
 	if (in->mask == 0) {
 		return 0; /* nothing to do */
 	}
-	if (in->mask & ~FIDO_EXT_CRED_PROTECT) {
+	if (in->mask & ~(FIDO_EXT_HMAC_SECRET|FIDO_EXT_CRED_PROTECT)) {
```

Comment:
```suggestion
	if (in->mask & ~(FIDO_EXT_HMAC_SECRET | FIDO_EXT_CRED_PROTECT)) {
```

Reply:
Fixed in d2e2593247c46a2db909d19590d0b0349d3bdced. Thanks!